lightning/src/ln/channel.rs

```diff
@@ -909,7 +909,8 @@ impl<Signer: Sign> Channel<Signer> {
 		let opt_anchors = false; // TODO - should be based on features
 
 		let holder_selected_contest_delay = config.channel_handshake_config.our_to_self_delay;
-		let holder_signer = keys_provider.get_channel_signer(false, channel_value_satoshis);
+		let channel_keys_id = keys_provider.generate_channel_keys_id(user_id);
```

Comment:
completely unrelated to this PR, but that variable really shouldn't be called `user_id`

Reply:
obviously, `user_provided_channel_id` is a bit verbose, but at least it's accurate